### ISSUE TITLE
Using Plotly.Fx.hover in callbacks

### ISSUE THREAD

**Issue by dbuezas** [dbuezas/lovelace-plotly-graph-card#398]
### Discussed in https://github.com/dbuezas/lovelace-plotly-graph-card/discussions/396

<div type='discussions-op-text'>

<sup>Originally posted by **paradox460** March 19, 2024</sup>
I'm trying to make a stacked graph initiate hover events on _all_ graphs when you hover over one graph. I figure I can do this with an `on_hover` callback, and a function, but I don't see how I can use the `Plotly.Fx` module; it does not appear to be available when probing from a browser's `debug` call or just calling directly in the `$fn`

Basically I'm trying to recreate the effect seen https://plotly.com/javascript/hover-events/#coupled-hover-events, but for a stacked grid of plots</div>
